expression/builtin_string_test.go

```diff
@@ -766,3 +766,28 @@ func (s *testEvaluatorSuite) TestCharLength(c *C) {
 		c.Assert(r, testutil.DatumEquals, types.NewDatum(v.result))
 	}
 }
+
+func (s *testEvaluatorSuite) TestFindInSet(c *C) {
+	defer testleak.AfterTest(c)()
+
+	for _, t := range []struct {
+		str    interface{}
+		strlst interface{}
+		ret    interface{}
+	}{
```

Comment:
Add this test:
mysql> select find_in_set("","");
+--------------------+
| find_in_set("","") |
+--------------------+
|                  0 |
+--------------------+
1 row in set (0.00 sec)